esp/scm/ws_dali.ecm

```diff
@@ -0,0 +1,26 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+EspInclude(common);
+
+ESPservice [auth_feature("NONE"), //This declares that the method logic handles feature level authorization
+    version("1.00"), default_client_version("1.00"), exceptions_inline("./smc_xslt/exceptions.xslt")] WSDali
+{
```

Comment:
is this is a placeholder for now?
And you plan on adding various ESPmethod ..... definitions here?
e.g. the current generic get for export, will be replaced by an explicit e.g. DaliExport(DaliExportRequest, DaliExportResponse) ?

Reply:
Will add more ESPmethods here. This PR is the first step of the ESP Dali service. More ESPmethods will be added in (step by step). 
I may add DaliExport(DaliExportRequest, DaliExportResponse) after this PR. In this PR, the export only has a stream download function because the content of the Export could be huge for some environment.

Reply:
fair enough, it should stream.
I suspect could be engineered so that the stream the response is using here was a bridge/adapter to the stream toXML is writing to.

Reply:
I think we may build (in another PR) a new bridge/adapter for toXML (CSocketIOAdapter?). ESP should add the code to set IProperTree as a response content. When sending the response, call the toXML to stream out the response. @afishbeck any comment?